apis/v1/gateway_types.go

```diff
@@ -654,6 +654,17 @@ type GatewayInfrastructure struct {
 	// +optional
 	// +kubebuilder:validation:MaxProperties=8
 	Annotations map[AnnotationKey]AnnotationValue `json:"annotations,omitempty"`
+
+	// ParametersRef is a reference to a resource that contains the configuration
+	// parameters corresponding to the Gateway. This is optional if the
+	// controller does not require any additional configuration.
+	//
+	// This follows the same semantics as GatewayClass's `parametersRef`, but on a per-Gateway basis
```

Comment:
I think we need to describe how merging works if/when this is configured at both layers.

Reply:
Added